src/limbo/model/gp/kernel_lf_opt.hpp

```diff
@@ -59,7 +59,7 @@ namespace limbo {
                         // K^{-1} using Cholesky decomposition
                         Eigen::MatrixXd w = Eigen::MatrixXd::Identity(n, n);
 
-                        gp.matrixL().template triangularView<Eigen::Lower>().solveInPlace(w);
+                        // gp.matrixL().template triangularView<Eigen::Lower>().solveInPlace(w);
```

Comment:
Why would you remove this line?
When you use the cholesky decomp, you have to first solve your system according to L and the according to L^t. 
Or am I missing something?